### ISSUE TITLE
import pandas as pd import numpy as np import matplotlib.pyplot as plt import seaborn as sns from sklearn.preprocessing import MinMaxScaler from sklearn.preprocessing import LabelEncoder import matplotlib.pyplot as plt from sklearn.model_selection import train_test_split from sklearn import preprocessing, metrics from sklearn.neighbors import KNeighborsClassifier from sklearn.metrics import accuracy_score, classification_report, confusion_matrix from sklearn.model_selection import GridSearchCV from sklearn.model_selection import train_test_split from sklearn.neural_network import MLPClassifier from sklearn.metrics import accuracy_score, classification_report, confusion_matrix import tensorflow as tf from tensorflow.keras.models import Sequential from tensorflow.keras.layers import InputLayer, Dense, Dropout, BatchNormalization from tensorflow.keras.regularizers import l1 import matplotlib.pyplot as plt

### ISSUE THREAD

**Issue by Svinjr** [oliahry/deeplearning#1]
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
from sklearn.preprocessing import MinMaxScaler
from sklearn.preprocessing import LabelEncoder
import matplotlib.pyplot as plt
from sklearn.model_selection import train_test_split
from sklearn import preprocessing, metrics
from sklearn.neighbors import KNeighborsClassifier
from sklearn.metrics import accuracy_score, classification_report, confusion_matrix
from sklearn.model_selection import GridSearchCV
from sklearn.model_selection import train_test_split
from sklearn.neural_network import MLPClassifier
from sklearn.metrics import accuracy_score, classification_report, confusion_matrix
import tensorflow as tf
from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import InputLayer, Dense, Dropout, BatchNormalization
from tensorflow.keras.regularizers import l1
import matplotlib.pyplot as plt
